src/pages/AppConfigure.vue

```diff
@@ -60,23 +68,40 @@ async function updateSmartContractTransactions(app: AppConfig) {
     if (hasStorageLimitChanged || hasBandwidthLimitChanged) {
       store.commit('showLoader', 'Updating user limits in smart contract...')
       await setDefaultLimit(app.storage_limit, app.bandwidth_limit)
+      toast.success('User limits saved in blockchain')
     }
+  } catch (e) {
+    handleSmartContractErrors('User limits', e)
+  }
 
+  try {
     store.commit('showLoader', 'Updating social auth in smart contract...')
     await setClientIds(app.cred)
+    toast.success('Client IDs saved in blockchain')
+  } catch (e) {
+    handleSmartContractErrors('Client IDs', e)
+  }
 
+  try {
     if (
       app.wallet_type === WalletMode.UI &&
       currentConfig.wallet_type === WalletMode.NoUI
     ) {
       store.commit('showLoader', 'Enabling UI Mode in smart contract...')
       await enableUiMode()
+      toast.success('UI mode saved in blockchain')
     }
   } catch (e) {
-    console.error(e)
-  } finally {
-    store.commit('hideLoader')
+    handleSmartContractErrors('UI mode', e)
   }
+  store.commit('hideLoader')
+}
+
+function handleSmartContractErrors(type: string, error: unknown) {
+  toast.error(
+    `An error occurred while saving the ${type} in the blockchain. Please try again or contact support`
```

Comment:
[nitpick] If you are using full sentences, end with a period.

```suggestion
    `An error occurred while saving the ${type} in the blockchain. Please try again or contact support.`
```

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/98/commits/916109a4d4a283ba3556ea4e093722166478b003